### DISCUSSION TITLE
Does React Native SVG 15.5.0 Support All SVG Filters

### DISCUSSION THREAD

**Discussion by YOEL311** [software-mansion/react-native-svg#2400] [Q&A] [answered]
I noticed that in version 15.5.0 of the React Native SVG library, support for SVG filters was added. I’m curious if this version now fully supports all SVG filter properties or if there are still some filters or attributes that are not yet supported.

My goal is to ensure that the SVGs I use in React Native render exactly as they do in a browser. Can anyone confirm if this is fully achievable with the current implementation in v15.5.0?

**Comment by jakex7** [comment, marked as answer]
Hi @YOEL311, 
you can follow the progress of the filters implementation here: [Filters progress #2362](https://github.com/software-mansion/react-native-svg/issues/2362)
Many things should already look like on the web, but there may be differences when using unimplemented filters.

**Reply by YOEL311** [reply]
Thanks for the info! Aside from filters, are there any other SVG properties in React Native SVG that might not render the same as they do on the web?
